code/modules/mob/living/basic/ruin_defender/mimic/mimic.dm

```diff
@@ -286,29 +288,30 @@ GLOBAL_LIST_INIT(animatable_blacklist, typecacheof(list(
 	return ((isitem(target) || isstructure(target)) && !is_type_in_typecache(target, GLOB.animatable_blacklist))
 
 /mob/living/basic/mimic/copy/proc/CopyObject(obj/original, mob/living/user, destroy_original = FALSE)
-	if(destroy_original || check_object(original))
+	if(!destroy_original || !check_object(original))
```

Comment:
Is this logic correct? Do we _never_ copy something if we don't destroy the original?

Reply:
it is absolutely not

Reply:
I assume it should be 
```suggestion
	if(destroy_original && !check_object(original))
```